### ISSUE TITLE
Dojo connected but transaction aren't updated in Samourai wallet

### ISSUE THREAD

**Issue by salenjak** [mynodebtc/mynode#255]
Hi,

I have "successfully???" connected Samourai wallet with Dojo on Mynode.

Everything seems to be working. All lights are green in Networking->Connections inside Samourai wallet and Dojo is enabled and running on Mynode, but Samourai wallet won't update the transaction that I sent a few days ago. It shows 0 BTC balance and no transactions history.

What I have tried that didn't help?
Reinstalling DOJO.
Re-flashing SD card with Mynode image.
I have even created a new wallet paired to Dojo, send few hundreds sats, but also won't update transactions history.

To prove that everything is OK with my seed and passphrase I have exported Samourai's zpub to Electrum and it shows transaction and correct amount of BTC.
Also,  I have deleted the Samourai wallet and then restored it from the seed, but without using Dojo. In that case, the wallet is working normally.

Any ideas?

**Comment by ratatosk9** [comment]
I have the exact same issue. I reached out to Samourai on the 8th of March on Twitter, still waiting for a reply (they might not check their twitter msgs). I did use my seed/passphrase and imported it in Electrum, and as you said, all transactions is there. It's like the Samourai Wallet app does not unlock with the passphrase, just the seed words.

If you find a solution to this, please reach out. My guess is that many people have the same issue.

**Comment by salenjak** [comment]
Hi ratatosk9,

The only thing I have been able to repair is "Bad Request" error inside my DOJO // MAINTENANCE TOOL. Now it's OK and Dojo is synced to the latest block. 
The trick I did was to disable Electrum server prior reinstalling Dojo.
Also, I have tried to get some help on Twitter (https://twitter.com/bitdov/status/1240596701230104576) because for this guy reinstalling worked.

I don't know will you understand what to do from this page (https://code.samourai.io/ronindojo/RoninDojo/-/wikis/Troubleshooting-&-FAQ#additional-dojo-issues), but maybe someone here can explain in a few steps how can we test and resolve problems under number 2 & 3.
Number 1 issue is "bad request", and we have managed to solve this one by reinstalling Dojo when electrum server is disabled. But this still after "supposedly" successfully paired Samourai wallet & mynode Dojo doesn't show balance and latest transactions. 

For the end, I also have some doubts about minimum hardware requirements, because my rpi4 has only 2Gb of ram & I believe that samurai somewhere stated that minimum configuration for DOJO must have 4GB. Anyone?

**Comment by Perplex0** [comment]
I have ran into this error multiple times and I think I have found a plausible explanation for the issue.
Dojo has its built in indexer to help find addresses related to the xpubs of your wallet. The indexer initializes after dojo has synced the blocks and like the electrum initialization it takes quite some time. Dojo does not do a good job of letting you know that the indexer is still indexing and it cannot find your transactions until after the indexer has finished, so it seems like it is not working even though it is fully set up. If you just wait for the indexer to finish initializing (which takes a few hours) dojo will be able to find your transactions again and work fine. That is what worked for me anyways.
